openedx/features/idea/models.py

```diff
@@ -10,15 +10,16 @@
 
 from lms.djangoapps.onboarding.models import Organization
 from .helpers import upload_to_path
+from .constants import CITY_MAX_LENGTH, TITLE_MAX_LENGTH, OVERVIEW_MAX_LENGTH
 
 
 class Location(models.Model):
     country = CountryField()
-    city = models.CharField(max_length=255)
+    city = models.CharField(max_length=CITY_MAX_LENGTH)
 
     @property
     def location(self):
-        return '{city}, {country}'.format(city=self.city, country=self.country)
+        return '{city}, {country}'.format(city=self.city, country=self.country.name)
```

Comment:
Let's make sure that this supports unicoded accented characters in country name.

Reply:
It works because of following lines at the top of file

```
# -*- coding: utf-8 -*-
from __future__ import unicode_literals